packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -200,6 +239,15 @@ This column will trigger `invalidColumnConfiguration` on the table's validity st
     >
         Temperature (°C)
     </nimble-table-column-number-text>
+
+    <nimble-table-column-number-text
+        field-name="fileSize"
+        format="decimal"
+        decimal-maximum-digits="1"
+        unit-family="nimble-unit-byte"
+    >
+        File Size
+    </nimble-table-column-number-text>
```

Comment:
This should have the units element within it, right?

```suggestion
        File Size
        <nimble-unit-byte></nimble-unit-byte>
    </nimble-table-column-number-text>
```

Reply:
I wasn't planning it that way. Just like the icon mapping column, we can just dynamically create the unit element. We don't even need it in the DOM.

Reply:
A child element might be more appropriate here. Part of the problem with icons was that they render and they have their own configuration settings as a visual component that might not be relevant to table column configuration so we made the column configuration capture only what it cared about (and it may have been a bit of an overcorrection from the flexibility / complication of the template approach).

In this case these elements are just for configuration so it may be worth keeping as children. I think it's also worth diverging from the icon pattern because we found that added some complexities (you need to import components you don't actually use in DOM and we need a reliable / typesafe way to get the tag names in Blazor we didn't figure out yet).

I can't think of any other reason (beyond pattern consistency with icon column which has issues) that these elements used only for config shouldn't be actual DOM children here.

Reply:
Changed API to take `nimble-unit-<name>` as content of the column element.